app/graphql/types/extensions/map_to_digital_objects.rb

```diff
@@ -8,8 +8,8 @@ def resolve(object:, arguments:, context:)
       end
 
       def after_resolve(object:, value:, arguments:, context:, memo:)
-        raise GraphQL::ExecutionError, 'ToDigitalObjects can only be downstream of Paginate' unless value.is_a?(OpenStruct) && value[:page_info].is_a?(OpenStruct)
-        value[:nodes] = value[:nodes].map { |digital_object_record| ::DigitalObject::Base.find(digital_object_record.uid) }
+        raise GraphQL::ExecutionError, 'ToDigitalObjects can only be downstream of SolrSearch' unless value.is_a?(OpenStruct) && value[:page_info].is_a?(OpenStruct)
```

Comment:
@barmintor did you mean "MapToDigitalObjects" here? Or is the "Map" part not made visible to the client in the way that it's used?

Reply:
really just an oversight

Reply:
```suggestion
        raise GraphQL::ExecutionError, 'MapToDigitalObjects can only be downstream of SolrSearch' unless value.is_a?(OpenStruct) && value[:page_info].is_a?(OpenStruct)
```